core/build.gradle.kts

```diff
@@ -193,20 +202,43 @@ kotlin {
             }
         }
 
-        val jsMain by getting {
+        val commonJsMain by creating {
+            dependsOn(commonMain.get())
             dependencies {
-                api("org.jetbrains.kotlin:kotlin-stdlib-js")
                 api("org.jetbrains.kotlinx:kotlinx-serialization-core:$serializationVersion")
                 implementation(npm("@js-joda/core", "3.2.0"))
             }
         }
 
-        val jsTest by getting {
+        val commonJsTest by creating {
+            dependsOn(commonTest.get())
             dependencies {
                 implementation(npm("@js-joda/timezone", "2.3.0"))
             }
         }
 
+        val jsMain by getting {
+            dependsOn(commonJsMain)
+            dependencies {
+                api("org.jetbrains.kotlin:kotlin-stdlib-js")
+            }
```

Comment:
This dependency should not be needed, after rebase to master when #319 is merged

Reply:
Removed